### DISCUSSION TITLE
Find a way to categorise the companies.

### DISCUSSION THREAD

**Discussion by Saphall** [Know-Nepal/tech-companies#38] [General]
What about finding a way to categorize the companies based on how they are_known/work for. 

> Searching for company are based on one's interest actually. For eg. I am looking for `Cybersecurity` company in nepal or `Machine Learning` company in nepal rather than looking for a company by a particular name.  

If I could search ML based company and I find company listed e.g `Leapfrog Technology`, then this will cutout furthur research that has to be done on what the company is based. How about adding a solution to categorize them?

**Comment by bsantosh909** [comment]
Sounds like a good idea to me.
Adding the type of work/solution the companies provide can be a good way to categorize them.

We can add a new key in the JSON file; `categories` which will be a array type holding string values.

I think we should work on thinking the string values that can be added here though.
Personally I think we should have a predefined list of items (which can be expanded as necessary) like
- `webDevelopment`
- `machineLearning`
- `artificialIntelligence`
- `cyberSecurity`
and so on.

**Reply by Saphall** [reply]
Yup for now, we could add a key holding the array of values as you mentioned, unless someone comes with a better idea though. If so, then we can proceed to finalize the pre-defined categories that go to the array like you mentioned @TheLearneer .
